### PR TITLE
Kimbra singlenozzle

### PR DESCRIPTION
**Pull request by tonnico**
I try to cherry-pick. So somebody needs to check this.

### CODE REVIEW

**Comment by thinkyhead** [comment]
That's very comprehensive. That last commit broke something, and the code is looking for `ARRAY_BY_EXTRUDERS` again. You might want to make a new branch from that second commit, then see if you can get a clean merge from the latest Development branch.

**Comment by tonnico** [comment]
I missed some renames. This works now.

But please test this before merge. I can't test this.

**Comment by thinkyhead** [comment]
Looks good, no typos that I can see. I will merge this pretty soon. Trying to find a bed leveling bug at the moment…

**Comment by thinkyhead** [comment]
This is only failing Travis CI due to issues on their end.

File: Marlin/ultralcd_implementation_hitachi_HD44780.h [hunk 1]
```diff
@@ -442,175 +442,173 @@ Possible status screens:
...
-#  if EXTRUDERS > 1
-    tHotend = int(degHotend(1) + 0.5);
-    tTarget = int(degTargetHotend(1) + 0.5);
-    lcd.print(LCD_STR_THERMOMETER[0]);
-#  else//Heated bed
-    tHotend=int(degBed() + 0.5);
-    tTarget=int(degTargetBed() + 0.5);
-    lcd.print(LCD_STR_BEDTEMP[0]);
-#  endif
+    #if HOTENDS >
```
Comment (thinkyhead) [inline Marlin/ultralcd_implementation_hitachi_HD44780.h]: If `HOTENDS` is greater than... what? This needs a right operand.

```
/ultralcd_implementation_hitachi_HD44780.h:485:19: error: operator '>' has no right operand
```

**Comment by thinkyhead** [comment]
@Wurstnase This went a little out of date with all my screwing around with `extruder_offsets`. When you have a chance to merge and resolve this, also make sure any new references to `EXTRUDERS` (if any) are updated to `HOTENDS` too. Lots of bugs have been fixed recently, so this PR will be easier to include without worrying about how it interacts with any of those.

**Comment by tonnico** [comment]
@thinkyhead I will make an update of it.

**Pull request closed without merging**